float32 Total_Length
float32 Total_Width
float32 Total_Height

float32 Min_Z
float32 Max_Z

int32 Num_Steps

float32 Step_Length
float32 Step_Width
float32 Step_Height

float64 Plane_Alpha


float32[] Edge_Coordinates


float32[] Points_Coordinates

